test/OpenTelemetry.Extensions.Propagators.Tests/EventSourceTest.cs

```diff
@@ -5,7 +5,7 @@
 using OpenTelemetry.Tests;
 using Xunit;
 
-namespace OpenTelemetry.Instrumentation.SqlClient.Tests;
+namespace OpenTelemetry.Instrumentation.Propagators.Tests;
```

Comment:
This namespace doesn't make sense to me.

```suggestion
namespace OpenTelemetry.Propagators.Tests;
```

Reply:
I have made only partial fix. Missed that the Instrumentation is not correct.